src/renderer/component/fileActions/view.jsx

```diff
@@ -15,34 +15,35 @@ type Props = {
   openModal: ({ id: string }, { uri: string }) => void,
   claimIsMine: boolean,
   fileInfo: FileInfo,
-  vertical?: boolean, // should the buttons be stacked vertically?
 };
 
 class FileActions extends React.PureComponent<Props> {
   render() {
-    const { fileInfo, uri, openModal, claimIsMine, vertical, claimId } = this.props;
+    const { fileInfo, uri, openModal, claimIsMine, claimId } = this.props;
     const showDelete = fileInfo && Object.keys(fileInfo).length > 0;
 
     return (
-      <section className={classnames('card__actions', { 'card__actions--vertical': vertical })}>
+      <React.Fragment>
         {showDelete && (
-          <Button
-            button="alt"
-            icon={icons.TRASH}
-            iconColor="red"
-            label={__('Delete')}
-            onClick={() => openModal({ id: MODALS.CONFIRM_FILE_REMOVE }, { uri })}
-          />
+          <Tooltip onComponent body={__('Delete this file')}>
+            <Button
+              button="alt"
+              icon={icons.TRASH}
+              description={__('Delete')}
+              onClick={() => openModal({ id: MODALS.CONFIRM_FILE_REMOVE }, { uri })}
+            />
+          </Tooltip>
         )}
         {!claimIsMine && (
-          <Button
-            button="alt"
-            icon={icons.REPORT}
-            href={`https://lbry.io/dmca?claim_id=${claimId}`}
-            label={__('Report content')}
-          />
+          <Tooltip onComponent body={__('Report content')}>
```

Comment:
Not a blocker, but it'd probably be superior if tooltip could be a property of a button, rather than a component that wraps a button.

Reply:
Ooh that's a good idea.